src/AnimatedSprite.h

```diff
@@ -74,6 +85,8 @@ class AnimatedSprite :
     sf::Sprite              _sprite;
     sf::RectangleShape      _highlight;
 
+    bool                    _isHighlighted;
```

Comment:
Is there a reason to add the extra member instead of just setting the rect to size 0 like I did?

Reply:
I mean, I don't really care. I'm just wondering if there was a reason to change it.